hack/dev-push-hyperkube.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script will build and push the hyperkube image.
+# This is useful for rapidly testing changes in a cluster that is
+# powered by hyperkube. You can configure the cluster to use a
+# hyperkube tag and then use this script to iterate on that tag.
+# Testing the new image is then as easy as pulling the tag on the nodes
+# and rebooting the nodes so that kubelet and kubelet managed services
+# utilize the updated hyperkube image.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/build/common.sh"
+
+if [[ -z "${KUBE_DOCKER_REGISTRY:-}" ]]; then
+	echo "KUBE_DOCKER_REGISTRY must be set"
+	exit -1
+fi
+if [[ -z "${KUBE_DOCKER_OWNER:-}" ]]; then
+	echo "KUBE_DOCKER_OWNER must be set"
+	exit -1
+fi
+if [[ -z "${KUBE_DOCKER_VERSION:-}" ]]; then
+	echo "KUBE_DOCKER_VERSION must be set"
+	exit -1
+fi
+
+kube::build::verify_prereqs
+kube::build::build_image
+kube::build::run_build_command hack/build-go.sh cmd/hyperkube
+
+REGISTRY="${KUBE_DOCKER_REGISTRY}/${KUBE_DOCKER_OWNER}" \
+VERSION="latest" \
```

Comment:
This should be $KUBE_DOCKER_VERSION rather than hard coded to "latest". Will fix this asap.